Makefile

```diff
@@ -37,17 +37,18 @@ build-base-image: download-csm-common
 	@echo "Base image build: SUCCESS"
 	$(eval BASEIMAGE=mpst-ubimicro:latest)
 
-.PHONY: docker
-docker: build-base-image
-	docker build -t csm-metrics-powerstore -f Dockerfile --build-arg BASEIMAGE=$(BASEIMAGE) --build-arg GOIMAGE=$(DEFAULT_GOIMAGE) .
+# Pre-requisites: RHEL, buildah, podman
+.PHONY: podman
+podman: build-base-image
```

Comment:
You may want to have an alias for the docker target to call the podman build. Unless you really want to eliminate docker from the thinking.

Reply:
We want to remove the docker target as we are seeing some issues with buildah and docker compatibility while building images.